src/project/.lisa/rules/shared/git-rules.md

```
@@ -67,7 +69,14 @@ lisa pr create --issue 40 --issue 41
 
 ### Update Ticket Status
 
-Use the `/jira` skill to transition tickets to Code Review:
+Use Linear MCP to transition tickets to In Review:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)